### ISSUE TITLE
Phpunit key broken in ubuntu 

### ISSUE THREAD

**Issue by bedus-creation** [calebporzio/better-phpunit#89]
It was working before 2 days ago, Now its says no task found.
I am with vscode vesion =>
1.30.2
61122f88f0bf01e2ac16bdb9e1bc4571755f5bd8
x64

**Comment by walkero-gr** [comment]
I have the very same issues on MacOS. The thing is that there is no info at the extension page on how to configure that.

**Comment by bedus-creation** [comment]
> I have the very same issues on MacOS. The thing is that there is no info at the extension page on how to configure that.



> I have the very same issues on MacOS. The thing is that there is no info at the extension page on how to configure that.
And I started using https://github.com/elonmallin/vscode-phpunit Everything works fine, but no color teminal in vscode.

**Comment by walkero-gr** [comment]
bazed on @calebporzio  reply here https://github.com/calebporzio/better-phpunit/issues/52#issuecomment-462057076

> If the package broke for you, you are either:
> A) On a NEWER version of code and HAVEN'T updated the extension
> B) On an OLDER version of code but HAVE updated the extension
> 
> The proper versions that should be working are:
> VS Code: v1.31.0 or above
> Better PHPUnit: 1.3.2

I guess the same thing happens for 1.3.3. In my case it happens to be at VSCode v1.30.2, so I guess thats the reason it doesn't work for me. What I did was to fork the project and checkout to version 1.2.2 commit

**Comment by calebporzio** [comment]
Hey, not wrapping my head around this at the moment - but let me know if there's something I can do to help others with this.

**Comment by walkero-gr** [comment]
I think that a solution to help people that can't update VSCode, is to release version 1.2.2 in a VSIX file. This way they can go back to a previous version.

It would be nice to have every version released as a VSIX file under releases here, in github.

**Comment by calebporzio** [comment]
Yeah, I can do that - I'll throw a VSIX file in there at least for 1.2.2

**Comment by calebporzio** [comment]
K, I put the 1.2.2 package AND the current one here: https://github.com/calebporzio/better-phpunit/tree/master/packaged_releases

**Comment by Chris1904** [comment]
Same issue on macOS (10.15.2) with VSCode (1.41.1)
`No task to run found. Configure Tasks...`

Other PHPUnit plugins seem to work properly, so sad haha

**Comment by calebporzio** [comment]
Sorry to everyone experiencing this issue.

I cannot replicate it, therefore I cannot investigate it.

I'm open to attempts at replicating, or ideally someone who is experiencing the issue, pulling down the repo and contributing the fix.

Thanks!

**Comment by shaneoliver** [comment]
This happened to me after I accidentally disabled TypeScript Auto Detect. Which stops the task runner
Check your settings.json file and remove the following line if it's there or set to true (default)
"task.autoDetect": "off",
